ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -236,19 +236,19 @@
       #end
       : $doc.display('participants')
       ## groups
-      ; &lt;label for="Meeting.Code.MeetingClass_0_groups"&gt;$!services.icon.render('group') $services.localization.render('contrib.meeting.field.groups')&lt;/label&gt;##
-      #if ($xcontext.action == 'edit')
+      #if ($xcontext.action == 'edit' || $doc.getValue('groups') != '')
+        ; &lt;label for="Meeting.Code.MeetingClass_0_groups"&gt;$!services.icon.render('group') $services.localization.render('contrib.meeting.field.groups')&lt;/label&gt;##
         (% class="xHint" %)$services.localization.render('')
```

Comment:
This is rendering an empty string, is there any translation associated to that label?